### DISCUSSION TITLE
Tables

### DISCUSSION THREAD

**Discussion by krkk** [gsantner/markor#1666] [Q&A] [answered]
Maybe in next update you add support to tables?

Example:

| Hours | Monday | Thursday | Friday |
|-|-|-|-|
| **8:00 - 8:45** | pe | english| maths |
| **8:55 - 9:40** | biology | geography | ict |

**Comment by gsantner** [comment, marked as answer]
Markor supports tables since long time. In next update (Markor>=v2.1.7) there will be textaction button to make it easy to create tables.

See video for #338 :
 ![video_2019-10-27_14-16-37](https://user-images.githubusercontent.com/6735650/67635210-38ae5500-f8bc-11e9-8ca0-6e45919bf329.gif)
